File: fizzbuzz/FizzBuzz.cs
public class Hello{
    public static void Main() {
        FizzBuzz.Run();
    }
}

public class FizzBuzz {

    public static void Run() {
        foreach(int i in System.Linq.Enumerable.Range(1, 100))
        {
            if (i % 15 == 0) {
                System.Console.WriteLine("FizzBuzz");
            } else if (i % 3 == 0) {
                System.Console.WriteLine("Fizz");
            } else if (i % 5 == 0) {
                System.Console.WriteLine("Buzz");
            } else {
                System.Console.WriteLine(i);
            }
        }
    }
}

File: fizzbuzz/FizzBuzz.scala
object Main extends App{

    def run(i: Int) = i match {
      case i if i % 15 == 0 => "FizzBuzz"
      case i if i % 3 == 0 => "Fizz"
      case i if i % 5 == 0 => "Buzz"
      case _ => i
    }
    
    (1 to 100).map(run).foreach(println)
}
